docs/organizations/managing-security-and-risk.md

```diff
@@ -445,3 +445,17 @@ Each Codacy issue reported by Security and risk management belongs to one of the
 | **Visibility**                     | Logging should always be included for security events to better allow attack detection and help defend against vulnerabilities.                                                                                  |
 | **XSS**                            | Cross-Site Scripting (XSS) attacks inject malicious client-side scripts into trusted websites that are visited by the end users.                                                                                 |
 | **Other**                          | Other language-specific security issues.                                                                                                                                                                         |
+
+## Scan types
+
+Each finding detected in an organization's repository is also categorized by **Scan type**, indicating the specific source or method used to detect the finding. This information helps you understand the origin of the finding and the context in which the underlying issue was detected.
+
+The following table lists the available scan types and their descriptions:
```

Comment:
@codacy/tarot I'm limiting the list to what we have as confirmed for now. I [removed the upcoming ones](https://github.com/codacy/docs/pull/2128/commits/6ad8b0edee86d820942d509920c4015242e65d4e) for now; open to including some/all of them, on condition that we're *really* sure they're upcoming. Wouldn't want to set the wrong expectations, especially in docs.